.github/workflows/artifacts-build.yml

```diff
@@ -63,7 +68,17 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: doc
-        args: --workspace --release --all-features --no-deps
+        # Exclude tool and derive crates
+        args: >
+          --workspace --release --all-features --no-deps --lib 
+          --exclude icu_benchmark_macros
+          --exclude icu_ffi_coverage
+          --exclude icu_provider_macros
+          --exclude tutorials-test
+          --exclude databake-derive
+          --exclude yoke-derive
```

Comment:
question: why not derive crates?

They do get documented, though for some of them we override their docs in reexports

Reply:
I guess it does save time since we don't have any non-fully-reexported derives

Reply:
Cleans up the sidebar as well